webapp/i18n/messages.en.json

```diff
@@ -0,0 +1,158 @@
+  "No": "No",
+  "Submit": "Submit",
+  "Save": "Save",
+  "Skip": "Skip",
+  "Cancel": "Cancel",
+  "OK": "OK",
+  "Show attributes": "Show attributes",
+  "Ash fall": "Ash fall",
+  "Lahar flows": "Lahar flows",
+  "Lava flows": "Lava flows",
+  "query attribute": "query attribute",
+  "attribute": "attribute",
+  "value": "value",
+  "min": "min",
+  "max": "max",
+  "error:file upload": "Please choose a file to upload.",
+  "error:draw point": "Please draw a point using the circlemarker drawing tool.",
+  "error:draw polygon": "Please draw a polygon using the polygon drawing tool.",
+  "error:number": "Please enter a numeric value.",
+  "error:name": "Please enter a name.",
+  "error:form numbers": "Please enter valid numbers in the fields.",
+  "Server error": "Server error",
+  "Unknown error": "Unknown error",
+  "User manual": "User manual",
+  "Select result to be displayed": "Select result to be displayed",
+  "Table description": "Table description",
+  "Column description": "Column description",
+  "Non-numeric columns are filtered out": "Non-numeric columns are filtered out",
+  "OSM Standard style": "OSM Standard style",
+  "OSM Humanitarian style": "OSM Humanitarian style",
+  "Basemap": "Basemap",
+  "Waterways": "Waterways",
+  "Roads": "Roads",
+  "Buildings": "Buildings",
+  "Basemap boundary": "Basemap boundary",
+  "Current selection": "Current selection",
+  "Time map": "Time map",
+  "Start point": "Start point",
+  "Via point": "Via point",
+  "Affected area": "Affected area",
+  "Road-level time map": "Road-level time map",
+  "Latacunga thematic maps": "Latacunga thematic maps",
+  "Elevation map": "Elevation map",
+  "Administrative units": "Administrative units",
+  "Population density": "Population density",
+  "Population density raster": "Population density raster",
+  "Greenhouses": "Greenhouses",
+  "Farms and orchards": "Farms and orchards",
+  "Schools": "Schools",
+  "Doctors and dentists": "Doctors and dentists",
+  "Hospitals and clinics": "Hospitals and clinics",
+  "Building floors": "Building floors",
+  "Latacunga roads": "Latacunga roads",
+  "Cotopaxi roads": "Cotopaxi roads",
+  "Lighting": "Lighting",
+  "Air distribution section": "Air distribution section",
+  "Underground distribution section": "Underground distribution section",
+  "Agriculture main crops": "Agriculture main crops",
+  "Land use and land cover": "Land use and land cover",
+  "Productive infrastructure": "Productive infrastructure",
+  "Producer associations": "Producer associations",
+  "Markets and squares": "Markets and squares",
+  "Safe points": "Safe points",
+  "Evacuation routes": "Evacuation routes",
+  "Early warning sirens": "Early warning sirens",
+  "Latacunga volcanic threats": "Latacunga volcanic threats",
+  "Affected areas": "Affected areas",
+  "Volcano lahar flow": "Volcano lahar flow",
+  "Cotopaxi ash fall": "Cotopaxi ash fall",
+  "Vulnerability": "Vulnerability",
+  "Map legend": "Map legend"
+}
```

Comment:
Should we better categorize the messages?  Like the following:
```
{ 
    ''message": { 
        "add_location.1": "...." , 
        "add_location.2": "...."
    }, 
    "button": {
        "Cancel": "...", 
        "Skip": "..."
    }
}
```
This will make it more readable when using them in the scripts.

Reply:
I think we should keep the files as simple as possible, to ease the work for the translators. But we could create multiple files as well, e.g. one for each module...

Reply:
That could be one approach. But I believe we need a structure to make it still maintainable, for example,  when the list grows very long and identical text contents appear in multiple UI elements.

Reply:
Hmm, I don't see how it would get so much harder to maintain. Even if a text appears multiple times, one translation is still enough.